docs/edge.md

```diff
@@ -186,6 +188,10 @@ An example of its output:
 | APPC/BTC  |      -0.02 |       0.44 |                2.28 |                   1.27 |         0.44 |                       25 |                       43 |
 | NEBL/BTC  |      -0.03 |       0.63 |                1.29 |                   0.58 |         0.44 |                       19 |                       59 |
 
+Edge produced the above table by comparing ``calculate_since_number_of_days`` to ``minimum_expectancy`` to find ``min_trade_number``. Historical information based on the config file. The time frame Edge uses for its comparisons can be further limited by using the ``--timeframe`` switch.
```

Comment:
I think what you mean with this is to limit the timerange, not the timeframe (timeframe = 5m - timerange would be "january-march" ...)
Unfortunately, the 2 words are very similar, so we need to be carefull to use them correctly.

```suggestion
Edge produced the above table by comparing `calculate_since_number_of_days` to `minimum_expectancy` to find `min_trade_number` historical information based on the config file. The timerange Edge uses for its comparisons can be further limited by using the `--timerange` switch.
```

Reply:
You are correct! My bad. Thanks for catching that.